config/dist/deployment-plans.yaml

```diff
@@ -195,7 +195,7 @@ application_orchestration:
               class: ConductorAppOrchestration\Deploy\Command\MakeBuildCurrentCommand
             reset-opcache:
               conditions: [code]
-              command: curl -sO http://gordalina.github.io/cachetool/downloads/cachetool.phar && php71 cachetool.phar opcache:reset
+              command: curl -sOL https://gordalina.github.io/cachetool/downloads/cachetool.phar && php cachetool.phar opcache:reset
```

Comment:
Changes confirmed:

- http to https
- L argument to curl to follow redirects (to reduce the impact of a change like this again in the future)
- `php71` changed to `php`. `php` was a relic of when we had to run php 5.6 side by side with php 7.x